vNext/src/OrleansAzureUtils/OrleansAzureUtils.csproj

```diff
@@ -0,0 +1,42 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup Label="Configuration">
+    <DefineConstants>NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard1.5</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <Compile Include="**\*.cs" />
+    <EmbeddedResource Include="**\*.resx" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\Build\GlobalAssemblyInfo.cs" />
+    <Compile Include="..\..\..\src\OrleansAzureUtils\**\*.cs" Exclude="..\..\..\src\OrleansAzureUtils\Properties\*.cs" />
```

Comment:
Maybe it's not going to affect this assembly because it doesn't have build-time codegen, but there are other C# files generated by the .NET tooling (I don't know their purpose) which should be excluded. You can exclude `..\..\..\src\OrleansAzureUtils\obj\**\*.cs;..\..\..\src\OrleansAzureUtils\bin\**\*.cs` to get rid of them.